packages/fxa-settings/src/pages/ResetPassword/ResetPasswordConfirmed/index.tsx

```diff
@@ -18,6 +19,10 @@ const ResetPasswordConfirmed = ({
   const [serviceName, setServiceName] = useState<string>();
   const [isSync, setIsSync] = useState<boolean>();
 
+  useEffect(() => {
+    GleanMetrics.resetPassword.createNewSuccess();
```

Comment:
Should this instead be createNewSuccessView (`password_reset_create_new_success_view`) for the front end event? 

I see from the Glean taxonomy that we should also be adding a backend ping for the password reset success (`password_reset_create_new_success`) in CompleteResetPassword's `onSubmit` - will that be done in this PR or a separate one?

Reply:
huh looking at the jiras for this branch I should've added the `password_reset_create_new_success` backend event not this.  Not sure what was I doing.  😬 

(The frontend `password_reset_create_new_success_view` was added to the sheet recently; I didn't know about those new rows until Friday afternoon last week.)